lib/Driver.js

```diff
@@ -139,9 +138,21 @@ module.exports = class Driver {
    *
    * @returns {string} Returns the key's id.
    */
-  async computeKeyId({key}) {
+  computeKeyId({key}) {
```

Comment:
Is this "Driver" class implementing a common interface? I suspect it is -- and if other implementations use `async`, that shouldn't be removed here.

Reply:
No idea... @dmitrizagidulin, thoughts?